packages/firebase_app_installations/firebase_app_installations/example/android/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.3.50'
+  ext.kotlin_version = '1.6.10'
```

Comment:
I don't see a single kotlin file in `packages/firebase_app_installations/firebase_app_installations/android/src/main`. Why is that needed?

Reply:
Build failed without updating oddly enough.

Reply:
I think it should be removed altogether 🤔 Since we don't use Kotlin, we are probably bundling unneeded stuff with this?